sunpy/instr/fermi.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import numpy as np
+import copy
+import math
+import urlparse
+import urllib, urllib2
+import tempfile
+import datetime
+import os
+import matplotlib.pyplot as plt
+
+from bs4 import BeautifulSoup
+import astropy.units as u
+from astropy.utils.compat.odict import OrderedDict
+from astropy.coordinates import Angle, Longitude, Latitude
+
+from sunpy.time import parse_time, TimeRange
+from sunpy import sun
+from sunpy.io.fits import fits
+
```

Comment:
you need to define an `__all__` list which contains the names of the public functions in this file.